test/ModuleInterface/if-configs.swift

```diff
@@ -85,6 +85,25 @@ public func hasClosureDefaultArgWithSinglePoundIf(_ x: () -> Void = {
 }) {
 }
 
+// CHECK: func hasClosureDefaultArgWithMultilinePoundIfCondition(_ x: () -> Void = {
+// CHECK-NOT: #if (
+// CHECK-NOT:   false && !true
+// CHECK-NOT: )
+// CHECK-NOT: print("should not appear")
```

Comment:
Hm, I think this test case it not testing the code change.
The change affects only for `getActiveClause()`, and this block is not active...

Reply:
I meant this test case should pass even without the change.

Reply:
@rintaro You're right, I pushed before testing locally 😅 They should pass now.